README.md

```diff
@@ -14,14 +14,18 @@ more like other UNIX tools.
 
 ## Installing RGBDS (UNIX)
 
-RGBDS requires libpng and pkg-config to be installed.
+RGBDS requires gcc, binutils, yacc, lex, libpng and pkg-config to be installed.
```

Comment:
Hmm… I'd rather not mandate GCC – for example, I'd prefer to compile using Clang.

Also, are we actually using `binutils` at the moment? I've tried a Find-Replace on `master` and couldn't find anything, and the only references to it in this PR are here and the installation example below.